packages/playground/package.json

```diff
@@ -28,8 +29,8 @@
         "react-router-dom": "^6.4.2",
         "tailwind-merge": "^2.3.0",
         "tailwindcss-animate": "^1.0.7",
-        "viem": "^1.18.2",
-        "wagmi": "^1.4.12",
+        "viem": "2.x",
+        "wagmi": "^2.10.9",
```

Comment:
the latest wagmi has some breaking changes, are we able to go with two different versions?

Reply:
I would like to use our playground with the latest version of all the stuff.
Unfortunately, there are some difficulties around here.

> are we able to go with two different versions?

Since harmony has river inside, `syncpack` won't allow.

> wagmi has some breaking changes

I might be wrong here, but harmony auth library already supports wagmi 2.x, and we have a few uses of standalone wagmi 1.x, so maybe it's worth (and easy) to upgrade?

My purpose with this is that:
1. I don't want our users to check some example code in the playground and see outdated code 😅
2. Ensure that the sdk works with a good DX along all the new stuff happening in the ecosystem.